### DISCUSSION TITLE
I can't apply from Github Education

### DISCUSSION THREAD

**Discussion by alex-des-santos** [community/community#141016] [GitHub Education]
### Hi everyone,

When I try to apply with the data below:
![image](https://github.com/user-attachments/assets/1d20e103-11d3-4d26-b2a2-776f977e1b8d)

I receive the following error:
![image](https://github.com/user-attachments/assets/136af6ed-407b-4885-aee6-7a79736f709a)

Notice that the name of the university/campus changes, and I can't figure out why.

**Comment by austinbediako** [comment]
it means that go to your GitHub settings and then add a secondary email that should be your school's email 
1. Open settings 
![image](https://github.com/user-attachments/assets/afed738f-34a1-4127-8065-ce54d6da3df2)
2. locate the email section in the navigation 
![image](https://github.com/user-attachments/assets/92c50211-ce9f-49c1-81bd-32909b3e0fe3)
3. you will see a section which says add email address 
![image](https://github.com/user-attachments/assets/e69b935b-6b6a-4c07-ac27-05067a2da8df)
Add the email and do all necessary verification after it has been added redo the student developer pack package again it should work just fine

**Reply by alex-des-santos** [reply]
My university does not provide email addresses with their domain for students, only for professors. What can I do?

**Reply by austinbediako** [reply]
It means your school hasnt registered for the student yet 
reach out to my profile and contact me so that we can work something out 
who knows you might be the first to be a github campus expert at your school
